app.mjs

```diff
@@ -29,6 +30,21 @@ const chainIdParamSchema = {
  * @param {import("fastify").FastifyInstance} fastify
  */
 export async function server(fastify, opts) {
+  Sentry.init({
```

Comment:
Can you please run this file through `prettier`? Just making sure we have consistent styling